src/discord-cluster-manager/leaderboard_db.py

```diff
@@ -52,7 +52,7 @@ def connect(self) -> bool:
         """Establish connection to the database"""
         try:
             self.connection = (
-                psycopg2.connect(DATABASE_URL, sslmode="require")
+                psycopg2.connect(DATABASE_URL)
                 if DATABASE_URL
```

Comment:
why tho? This would disable ssl everywhere

Reply:
We disabled this while @alexzhang13  and I were trying to dev locally otherwise it throw errors. Maybe a better way is to have a local debug mode that does not require SSL.